### ISSUE TITLE
Add comparable metrics to W&B across batch sizes and resolutions

### ISSUE THREAD

**Issue by jder** [m2lines/Samudra#795]
When looking at W&B across different runs configured for different batch sizes, numbers of GPUs, and resolutions, we can only compare the x-axis of training step. Seems like it would be helpful to log things which are more comparable, like total examples trained on, or (approximately) total bytes trained on, or similar. 

@fomo-bot can you review what is typically done here and propose some metrics we could easily extract & log?

**Comment by jder** [comment]
@fomo-bot ok, can you make a PR with 1-6? I think wall seconds is implicitly available in W&B already but GPU-seconds sounds good. Your non-cumulative things sound good too. Let's hold off on making it the default for now.
